gradle.properties

```diff
@@ -48,6 +48,7 @@ stdlibUuidVersion=1.8.0
 # Stdlib Level 04
 stdlibAuthVersion=2.11.0
 stdlibDataJsondataVersion=0.1.0
+stdlibDataYamlVersion=0.1.0-20240802-131800-0659427
```

Comment:
You have to add this to the build.gradle file in order to pack this with the Distribution.

Reply:
I will update it